File: 1291-sequential-digits/1291-sequential-digits.cpp
class Solution {
public:
    vector<int> sequentialDigits(int low, int high) {
        vector<int> result;
        
        // Try every possible starting digit
        for (int start = 1; start <= 9; ++start) {
            int num = start;
            
            // Append digits sequentially
            for (int next = start + 1; next <= 9; ++next) {
                num = num * 10 + next;

                if (num >= low && num <= high) {
                    result.push_back(num);
                }
            }
        }
        
        sort(result.begin(), result.end());
        return result;
    }
};

File: 1291-sequential-digits/1291-sequential-digits.py
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        result = []

        for start in range(1, 10):
            num = start

            for next_digit in range(start + 1, 10):
                num = num * 10 + next_digit
                
                if low <= num <= high:
                    result.append(num)

        result.sort()
        return result